inst/private/doctest_collect.m

```diff
@@ -64,16 +64,30 @@
   files = dir('.');
   for i=1:numel(files)
     f = files(i).name;
-    if strcmp(f, '.') || strcmp(f, '..') || strcmpi(f, 'private')
-      % skip ., .., and private folders (TODO)
-      continue
-    end
-    if (f(1) == '@')
-      % strip the @, prevents processing as a directory
-      f = f(2:end);
-    elseif (~ recursive && exist(f, 'dir'))
-      % skip directories
-      continue
+    if (exist(f, 'dir'))
+      if (strcmp(f, '.') || strcmp(f, '..'))
+        % skip "." and ".."
+        continue
+      elseif (strcmp(f(1), '@'))
+        % strip @ to prevent processing as a directory
+        f = f(2:end);
+      elseif (~ recursive)
+        % skip all directories
+        continue
+      elseif (strcmp(f(1), '.'))
+        fprintf(fid, 'Ignoring hidden directory "%s"\n', f)
```

Comment:
Ok, I'll do that.  What about the "descending into foo" message?  It can be misleading without a corresponding "done with foo"...  I could try indenting two spaces for each level of depth...